Dockerfile

```diff
@@ -10,20 +10,20 @@ COPY . .
 
 RUN CGO_ENABLED=0 go build -v -o /bin/gotty .
 
-FROM golang:1.17 as usql
+FROM golang:1.19.1-bullseye as usql
 
 WORKDIR /go/src/github.com/tidbcloud/gotty
 
 COPY tools tools
 RUN cd tools/usql && go mod download
 RUN ./tools/usql/install.sh
 
-FROM gcr.io/distroless/static:nonroot
+FROM debian:bullseye-slim
```

Comment:
why change the base image?

Reply:
When I tried to figure out what actually happened, I had to use tcpdump to capture network traffic. Distroless made it very hard to do that.

Reply:
It's not our code, next time when we see some wired issue tools like tcpdump/strace are very important to diagnose such a blackbox.

Reply:
`gotty` can be directly reached by the end user. So we remove the `shell` to avoid some security issues.

Reply:
Make sense, let's revert this.